cmd/dist/get.go

```diff
@@ -11,13 +11,11 @@ import (
 )
 
 var getCommand = cli.Command{
-	Name:      "get",
-	Usage:     "get the data for an object",
-	ArgsUsage: "[flags] [<digest>, ...]",
-	Description: `Display the paths to one or more blobs.
-	
-Output paths can be used to directly access blobs on disk.`,
-	Flags: []cli.Flag{},
+	Name:        "get",
+	Usage:       "get the data for an object",
+	ArgsUsage:   "[flags] [<digest>, ...]",
+	Description: "Display the image object.",
```

Comment:
This is not what this does. It outputs a blob from the content addressable storage.